src/bridge/BridgeBase/index.ts

```diff
@@ -44,7 +44,7 @@ export class BridgeBase extends LitElement {
   //  2nd: inLogging - sets this.bLogging: false if not provided.
   constructor(inDebug = false, inLogging = false) {      
       super();
-
+      LitElement.disableWarning?.('change-in-update');
```

Comment:
Can you please leave a comment here explaining why we're doing this and noting that we should continue to investigate if there are updates to Lion that can get the warnings to go away. The doc for this indicates that the warnings may only be suppressed in development builds and I'd like to keep working on this as time allows. Thanks!